packages/wonder-blocks-dropdown/src/components/action-menu.tsx

```diff
@@ -203,14 +203,18 @@ export default class ActionMenu extends React.Component<Props, State> {
                 };
                 // @ts-expect-error [FEI-5019] - TS2345 - Argument of type 'ReactChild | ReactFragment | ReactPortal' is not assignable to parameter of type 'ReactElement<any, string | JSXElementConstructor<any>>'.
             } else if (OptionItem.isClassOf(item)) {
+                const selected = selectedValues
+                    ? selectedValues.includes(value)
+                    : false;
                 return {
                     ...itemObject,
                     populatedProps: {
                         onToggle: this.handleOptionSelected,
-                        selected: selectedValues
-                            ? selectedValues.includes(value)
-                            : false,
+                        selected,
                         variant: "check",
+                        role: "menuitemcheckbox",
```

Comment:
This is where the main change is! When an ActionMenu is used, we set these attributes for OptionItem components. We set these attributes here since this should only apply to OptionItems in an ActionMenu (not when it is used in a Single/MultiSelect). 

The remaining changes make it so these attributes are applied/received as props